Tests/Unit/EmogrifierTest.php

```diff
@@ -1108,7 +1108,7 @@ public function emogrifyWithValidMediaQueryContainsInnerCss($css)
     public function emogrifyForHtmlWithValidMediaQueryContainsInnerCss($css)
     {
         $html = $this->html5DocumentType . PHP_EOL . '<html><style type="text/css">' . $css
-            . '</style><h1></h1></html>';
+            . '</style><h1></h1><p></p></html>';
```

Comment:
Because the new behavior removes the media query CSS if it doesn't match any selectors. We need a <p></p> so the media queries stay in the document.